.tekton/basic-component-pipeline.yaml

```diff
@@ -101,7 +101,7 @@ spec:
     value: $(tasks.clone-repository.results.commit)
   - description: ""
     name: JAVA_COMMUNITY_DEPENDENCIES
-    value: $(tasks.build-container.results.JAVA_COMMUNITY_DEPENDENCIES)
+    value: $(tasks.build-container-amd64.results.JAVA_COMMUNITY_DEPENDENCIES)
```

Comment:
Same for the post-build tasks as well

Reply:
We don't have JAVA dependencies, so it doesn't apply to this particular line. The prefetch of dependencies does copy the go dependencies, but those are fine because the same code is used for all archs AFAIK.

The post-build tasks were directly copied from the examples we were given so 🤷🏻‍♂️ , I need the opinion of experts on this one.

Reply:
Yes, we don't have Java dependencies so following the example like you did should be fine.

For post-tasks, I suppose, the ones that deal with container scanning seem like need one container, not all. There's no conceptual difference between giving them `amd64` container and giving them manifest because `amd64` one will be selected from the manifest according to the arch of the node where the code executes.

I don't know if these tasks are actually able to work with manifests or there's some limitation that they need only actual images. Using manifests seems cleaner so I wouldn't be at all opposed if you try (but you don't have to).

Reply:
Oh, I see that you actually did.